charts/prometheus-rabbitmq-exporter/values.yaml

```diff
@@ -71,9 +71,69 @@ prometheus:
     namespace: []
     metricRelabelings: []
     relabelings: []
+
+## Custom PrometheusRules to be defined
+## The value is evaluated as a template, so, for example, the value can depend on .Release or .Chart
+## ref: https://github.com/coreos/prometheus-operator#customresourcedefinitions
+prometheusRule:
+  enabled: false
+  additionalLabels: {}
+  namespace: ""
   rules:
-    enabled: false
```

Comment:
this looks like as we're changing this to have the rules enabled by default?
i would keep it disabled, to not change default behaviour of the chart.
this and the other changed variables would be major version change too.

Reply:
I have made changes so that the enabling of rules is still controlled through `values.prometheus.rules`. Then, the original rules have been moved to `values.prometheus.rules.additionalRules`.